docs/LocalDevelopmentGuide.md

```diff
@@ -76,82 +89,79 @@ We send our logs to CloudWatch for easier storage and querying.
 
-## 5: Set Up Config
-
-Our repo already contains `sample-env`, the main environmental configuration
-file for the entire app, as well as the `credentials/` directory, where you
-will put credential files like the Github app private key. The file is split
-into section. There is a general section (which should be the top bit), a
-section on everything slack related, a section on Github and Github apps,
-and a section on AWS. [Please read the section about the configuration
-system.][config]
+CloudWatch integration is not currently supported in this manner.
 
-### 5.1: Set up Github App and organization
+### 5: Set up a GitHub App and organization
 
-Register Rocket 2 as a Github App under an appropriate testing organization
-(our team has one of these set up already). Make sure to install the Github App
-to the organization in addition to registering it.
+Create a Rocket 2 Github under an appropriate testing organization. Make sure
+to install the GitHub App to the organization in addition to registering it.
+All this can be done from a GitHub organization's Settings > GitHub Apps page.
 
-Under "Private keys", click "Generate a new private key". This will generate
-and allow you to download a new secret key for Rocket 2. Save this to the
-`credentials/` directory as `github_signing_key.pem` - it should already be in
-the PEM file format, bracketed by:
+In the GitHub app's settings, go to "Private keys" and click "Generate a new
+private key". This will generate and allow you to download a new secret key for
+Rocket 2. Save this to the `credentials/` directory as `gh_signing_key.pem` -
+it should already be in the PEM file format, bracketed by:
 
 ```
 -----BEGIN RSA PRIVATE KEY-----
 ...
 -----END RSA PRIVATE KEY-----
 ```
 
-Authenticating Rocket 2 as a Github App and obtaining an access token for the
-Github API should be automated, once the signing key is available.
+Authenticating Rocket 2 as a GitHub App and obtaining an access token for the
+GitHub API should be automated, once the signing key is available. Refer to
+the [GitHub key configuration docs](Config.md#github-key) for the required
+permissions.
 
-After doing this, remember to put your ngrok HTTPS URL with `/webhook` appended
-at the end, into the "Webhook URL" box. After doing this, you must go to the
-app's "Permissions & Events" tab and set the following as Read & Write:
+After doing this, remember to put your tunneled HTTPS URL with `/webhook`
+appended at the end into the "Webhook URL" box. Refer to the
+[GitHub webhook configuration docs](Config.md#github-webhook-endpt) for the
+required subscriptions.
 
-- Organization members
+## 6: Set Up Configuration
 
-After doing so, please check the checkboxes below:
+Our repo already contains `sample-env`, the main environmental configuration
+file for the entire app, as well as the `credentials/` directory, where you
+will put credential files like the GitHub app private key.
```

Comment:
As above, we don't use the `credentials` directory.

Reply:
Will be used in #484 - it's handy as a place to put credentials in the meantime